File: FrontEsprit/SmartConseil-Front/src/app/PlanificationConseil/conseil/ConseilDTO.ts
import { Utilisateur } from "src/app/utilisateur/Utilisateur";

export class ConseilDTO {

    id!: number;
  date!: Date;
    heure!: string;

  duree!: string;
  description!: string;

  optionId!: number;
  classeIds!: number[];
  salleId!: number;
  presidentId!:number
raporteurId!:number
  etat!: boolean;
  deroulement!: string;
}

File: BackEsprit/SmartConseil-Back/microservices/microservicePlanification/src/main/java/tn/esprit/microserviceplanification/Entity/ConseilDTO.java
package tn.esprit.microserviceplanification.Entity;

import lombok.*;
import java.time.LocalTime;
import java.util.Date;

import java.util.List;


@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ConseilDTO {
    private Integer id;
    private Date date;
    private String duree;
    private String description;

    private Integer optionId;
    private List<Integer> classeIds;

    private LocalTime heure;

    private Integer salleId;

    private Long presidentId;
    private Long raporteurId;
    private String deroulement;
}
